alns/ALNS.py

```diff
@@ -1,17 +1,15 @@
 import logging
 import time
-from collections import defaultdict
-from typing import Callable, Dict, Iterable, List, Optional, Tuple
+from typing import Callable, Dict, List, Optional, Tuple
 
-import numpy as np
 import numpy.random as rnd
 
 from alns.Result import Result
 from alns.State import State
 from alns.Statistics import Statistics
 from alns.accept import AcceptanceCriterion
 from alns.stop import StoppingCriterion
-from alns.select import SelectionScheme
+from alns.select import OperatorSelection
```

Comment:
I'm not entirely sure about this name. I feel like it needs "Scheme" at the end, or some such thing.

Reply:
I changed it to `OperatorSelectionScheme`. It's verbose (a bit too verbose?) but it's very clear now.

Reply:
It is a bit long, but on the other hand: a user would only have to import this if they want to write their own. And for internal use it's all very clear again :D.